modules/troubleshooting-network-observability-after-installation.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * networking/network_observability/troubleshooting-network-observability.adoc
+
+:_content-type: PROCEDURE
+[id="configure-network-traffic-console_{context}"]
+= Configuring network traffic menu entry in the {prouct-title} console
+
+Manually configure the network traffic menu entry in the {product-title} console where network traffic menu entry is not connected to the network.
+
+.Prerequisites
+
+* You have installed {product-title} version 4.10+.
+
+.Procedure
+
+. Check if the `spec.consolePlugin.register` field is set to `true` by running the following command:
++
+[source,terminal]
+----
+$ oc -n netobserv get flowcollector cluster -o yaml
+----
++
+.Example output
+----
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  consolePlugin:
+    register: true
```

Comment:
Show the example output as if the spec was not registering the console so the user just follow the logic on how to update it:
```suggestion
    register: false
```

Reply:
Addressed